src/quickstart/deploy-an-app.md

```diff
@@ -1,26 +1,44 @@
-## Register an Admin User
+## Determine your host and hostname values
 
-The first user to register against Deis Workflow will automatically be given administrative privileges.
+For the rest of this example we will refer to a special variables called `$hostname`. Please choose one of the two methods for building your `$hostname`.
 
-If you installed Deis on GKE or AWS, Deis automatically creates a load balancer for the cluster. To get the IP of this load balancer, run `kubectl --namespace=deis describe svc deis-router`.
+#### Option 1: Standard Installation 
+
+For a standard installation you can build the hostname using public IP address and a wildcard DNS solution. Instead of setting up DNS records, this example will use `nip.io`. 
```

Comment:
Maybe:

"For a standard installation that includes deis-router, you can build the hostname using its public IP address and a wildcard DNS record."